core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackVolumeUpdater.java

```diff
@@ -10,10 +10,8 @@
     public void updateVolumeIfNecessary(PlaybackServiceMediaPlayer mediaPlayer, long feedId,
                                         VolumeAdaptionSetting volumeAdaptionSetting) {
         Playable playable = mediaPlayer.getPlayable();
-        boolean isFeedMedia = playable instanceof FeedMedia;
-        boolean isPlayableLoaded = isPlayableLoaded(mediaPlayer.getPlayerStatus());
 
-        if (isFeedMedia && isPlayableLoaded) {
+        if (playable instanceof FeedMedia) {
```

Comment:
Thanks for your commits, @ByteHamster. Perhaps this condition was a bit overcautious...

Reply:
Are you okay with my changes or do you notice anything that could lead to problems? If you are happy, we can finally merge this now :)

Reply:
Happy I am, no objections!